DevelopedCalculator.js

```diff
@@ -32,10 +31,11 @@ class DevelopedCalculator extends Calculator {
     onOperatorButtonClick(event) {
             // calling parent class's onOperatorButtonClick method
             super.onOperatorButtonClick(event);
-            // pushing the operator to the history array
-            this.history.push(this.operator);
+            // pushing current result and the operator to the history array
+            this.history.push(this.operator, this.currentResult);
```

Comment:
When I click on 
1 then + 

I should see 1 then + in the history element and to do that 
do I have to push the operator first or the current result to the history array?

Reply:
Yes now (with removing pushing the current result in on number button click method the operator should be later
Because I will push the current result in equal button

Thanks